axelrod/strategies/sequence_player.py

```diff
@@ -59,8 +65,12 @@ def __init__(self):
 
 
 class ThueMorseInverse(ThueMorse):
-    """A player who defects or cooperates according to the Thue-Morse sequence
-    (Inverse of ThueMorse)."""
+    """ A player who plays the inverse of the Thue-Morse sequence.
+
+    Names:
+
+    - ThueMorse: Original by Geraint Palmer
```

Comment:
You missed this one @Nikoleta-v3. Needs to be "Inverse Thue Morse". (Currently both the Thue Morse strategies have the same name).